AI/libs/llm/ollama.py

```
@@ -0,0 +1,49 @@
+        try:
+            response = requests.post(url, json=payload)
+            response.raise_for_status()
+            result = response.json()
+            return result.get("response", "")
+        except Exception as e:
+            print(f"[OllamaClient][Error] 텍스트 생성 실패: {e}")
+            return ""
+
+    def get_health(self) -> bool:
+        try:
+            res = requests.get(self.base_url)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)